### PR TITLE
xl-client changes in jenkinsfile

### PR DESCRIPTION
**Pull request by sagarkar-xl**
Description:

User Story: https://digitalai.atlassian.net/browse/ENG-4389
Modified Jenkins file added the xl-client changes to render and build and push docker image for alpine.

### CODE REVIEW

**Comment by karthiksonti24** [comment]
@sagarkar-xl this will work only for releases that we release from master. please come up with a solution that serves releases from maintenance branches too.

**Comment by sagarkar-xl** [comment]
Description:

User Story: https://digitalai.atlassian.net/browse/ENG-4389
Modified Jenkins file added the xl-client changes to render and build and push docker image for alpine.

**Comment by sagarkar-xl** [comment]
Description:

User Story: https://digitalai.atlassian.net/browse/ENG-4389
Modified Jenkins file added the xl-client changes to render and build and push docker image for alpine.

File: Jenkinsfile [hunk 1]
```diff
@@ -143,7 +143,7 @@ pipeline {
                                 }
 
                                 // Build Docker Image and push it
-                                sh "pipenv run ./applejack.py build --xl-version ${xlclient_LatestVersion} --download-source nexus --download-username ${NEXUS_CRED_USR} --download-password ${NEXUS_CRED_PSW}  --product xl-client  --target-os alpine --push --registry ${params.Registry}"
+                                sh "pipenv run ./applejack.py build --xl-version ${xlclient_LatestVersion} --download-source nexus --download-username xlict --download-password 3iZUEk76DjUo  --product xl-client  --target-os alpine --push --registry ${params.Registry}"
```
Comment (bashbers) [inline Jenkinsfile]: @sagarkar-xl I hope you have since then rotated this password? Otherwise your password is in plaintext hidden in your commithistory 😓

**Pull request closed without merging**